var/spack/repos/builtin/packages/highfive/package.py

```diff
@@ -29,10 +29,17 @@ class Highfive(CMakePackage):
     depends_on('hdf5 ~mpi', when='~mpi')
     depends_on('hdf5 +mpi', when='+mpi')
 
-    def cmake_args(self):
-        args = [
+    def common_cmake_args(self):
+        return [
             '-DUSE_BOOST:Bool={0}'.format('+boost' in self.spec),
             '-DHIGHFIVE_PARALLEL_HDF5:Bool={0}'.format('+mpi' in self.spec),
-            '-DUNIT_TESTS:Bool=false',
```

Comment:
This should be conditional: we want to run unit tests when `spec.satisfies('+test')`. That's how we run the CI normally.

Reply:
The problem is that the examples can't be built with MPI because HPE headers are broken. I did not disable the tests myself, so if you don't mind, I prefer not fixing that stuff in this PR because it promises not being trivial.